lib/Sema/DerivedConformance/DerivedConformanceDistributedActor.cpp

```diff
@@ -332,17 +332,24 @@ deriveBodyDistributed_invokeHandlerOnReturn(AbstractFunctionDecl *afd,
     metatypeVar->setImplicit();
     metatypeVar->setSynthesized();
 
+    // If the SerializationRequirement requires it, we need to emit a cast:
     // metatype as! <<concrete SerializationRequirement.Type>>
+    bool serializationRequirementIsAny =
+        metatypeParam->getInterfaceType()->getMetatypeInstanceType()->isEqual(
+            C.getAnyExistentialType());
```

Comment:
You can use `metatypeParam->getInterfaceType()->isAny()`

Reply:
[`eff8cc2` (#84119)](https://github.com/swiftlang/swift/pull/84119/commits/eff8cc2e7e5c25aae60ce6949b89419b2537627a)